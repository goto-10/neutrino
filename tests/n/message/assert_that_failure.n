import $assert;
do $assert:that(false);
