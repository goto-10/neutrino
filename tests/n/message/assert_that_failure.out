Error: %<condition: UncaughtSignal(escape)>
Info: --- backtrace ---
- leave.assert_failed()
- :fail()
- :that(false)
